components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/domain.jsp

```diff
@@ -16,7 +16,11 @@
   ~ under the License.
   --%>
 <%@ page import="org.owasp.encoder.Encode" %>
+<%@ page import="java.util.ResourceBundle" %>
 <%
+    String BUNDLE = "org.wso2.carbon.identity.application.authentication.endpoint.i18n.Resources";
+    ResourceBundle resourceBundle = ResourceBundle.getBundle(BUNDLE, request.getLocale());
+    String domain_unknown = resourceBundle.getString("domain.unknown");
```

Comment:
we don't normally use these type of naming conventions. It goes normally like "domainUnknown"